.github/workflows/linkchecker.yml

```diff
@@ -0,0 +1,12 @@
+name: Check Markdown links
+on: 
+  schedule: 
+    - cron: '* */24 * * *'
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: gaurav-nelson/github-action-markdown-link-check@1.0.12
+        with:
+          folder-path: "docs"
```

Comment:
If I do this, /README.md won't be included?

Reply:
Currently, it is only checking `docs` folder. Do you want it to check all markdown files in the repository or just include README.md file with docs folder?

Reply:
how about target the entire repo to include a spec doc in the x folder?

Reply:
As discussed offline, I'll let it to check all markdown files in the repository.